plane/src/typed_unix_socket/mod.rs

```diff
@@ -30,10 +30,7 @@ mod tests {
     use futures_util::future::join_all;
     use std::collections::HashSet;
     use std::env;
-    use std::ops::Deref;
     use std::path::PathBuf;
-    use std::sync::Arc;
-    use std::time::Duration;
     use tokio::spawn;
```

Comment:
nit: since you've added some `tokio::` to the spawns below, maybe do it for all of them for consistency? I'm not super opinionated on whether we use `tokio::spawn` or `spawn` as long as it's consistent within a file, but I lean towards `tokio::spawn` everywhere for clarity.